other/hell/demon

```diff
@@ -1,8 +1,13 @@
-**Demon** | Solo Miscellaneous - Hell Team
+**Demon** | Solo Killing - Hell Team
 __Basics__
 The Demon wins with the devil and joins a channel with the devil to communicate securely.
+Just like the devil the Demon can use the hell team's pool of souls.
 __Details__
-The Demon is usually the result of someone copying the devil for various reasons. This is because the devil is too powerful for there to be two. 
-Their win condition is the same as the Devil’s. The Demon starts out soulless.
+At the start of the game, the Demon may choose a role as a weak disguise. If the Demon doesn’t choose a role, they will show up as a citizen.
+
+The Demon is soulless and their soul is in the hell team's pool of souls.
```

Comment:
maybe mention the starting pool of souls?

Reply:
Did you miss this?

> The Demon is soulless and their soul is in the hell team's pool of souls.

Alternatively I might be misunderstanding what you're saying

Reply:
they are not in there the whole game if they get used

Reply:
Ah, I see.